cmd/lncli/cmd_open_channel.go

```diff
@@ -328,10 +336,19 @@ func openChannel(ctx *cli.Context) error {
 			return fmt.Errorf("unable to decode local amt: %v", err)
 		}
 		args = args.Tail()
-	default:
+	case !ctx.Bool("fundmax"):
 		return fmt.Errorf("local amt argument missing")
 	}
 
+	// Note:
```

Comment:
nit: no need for the explicit "Note" line, this is just a sanity check like others in this file.